tests/integration/standard/test_authentication.py

```diff
@@ -115,7 +115,7 @@ def test_auth_connect(self):
             finally:
                 cluster.shutdown()
         finally:
-            root_session.execute('DROP USER %s', user)
+            root_session.execute('DROP USER IF EXISTS %s', user)
```

Comment:
Why is this change here? It's not obvious for me and not explained in commit message

Reply:
I'll take it down, and give it a go.

but this failing in some cases, might mask the actual failure.
I think cause this is the first test in this module, starts before all of cluster is ready.

I also notices that it's not having `system_auth` replicated to all nodes

Reply:
meanwhile I've found what I think is the actual root cause, calls to ccm `cluster.start` without `wait_other_notice=True`, which can lead to unready cluster, and to test starting in a state that not all of the nodes are yet connected to each other